.github/workflows/publish-npm-package.yml

```diff
@@ -0,0 +1,58 @@
+name: Publish an NPM package
+
+on:
+  workflow_call:
+    inputs:
+      package_name:
+        required: false
+        type: string
+        default: ${{ github.event.repository.name }}
+      deploy_to_github_pages:
```

Comment:
@ChrisBAshton This is overkill but our Visual Editor does not have this configured yet. All other packages I looked at do, so if we want to reuse this workflow, we should include this step.

Reply:
Nice solution 👍